content/en/docs/reference/scheduling/config.md

```diff
@@ -147,10 +196,25 @@ that are not enabled by default:
   Service across nodes.
   Extension points: `PreFilter`, `Filter`, `Score`.
   
-## Multiple profiles
+### Multiple profiles
+
+`kube-scheduler` can be configured to
```

Comment:
nit (OK to merge without addressing this):
```suggestion
You can configure the `kube-scheduler` to
```

Reply:
Done